tasks/install.yml

```diff
@@ -11,14 +11,8 @@
   apt:
     name: "{{ item.name }}"
     state: "{{ item.state }}"
-  with_items: "{{ haproxy_dependencies }}"
+  with_items:
+    - "{{ haproxy_dependencies }}"
+    - "{{ haproxy_install | default('latest') }}"
```

Comment:
This seems incorrect

Reply:
Ans both `haproxy_dependencies` and `haproxy_install` are lists, so yuou probably need to do something like `haproxy_dependencies + haproxy_install`